networkx/algorithms/operators/binary.py

```diff
@@ -58,18 +59,19 @@ def union(G, H, rename=(None, None), name=None):
     # Union is the same type as G
     R = G.__class__()
     if name is None:
-        name = "union( %s, %s )"%(G.name,H.name)
+        name = "union( %s, %s )" % (G.name, H.name)
```

Comment:
@SanketDG I think it's not a pep8 issue. Rather it's re factoring code for Python 2/3.